api/bindings.h

```diff
@@ -110,10 +110,11 @@ typedef struct querier_t {
 } querier_t;
 
 typedef struct Querier_vtable {
-  int32_t (*query_external)(const querier_t*, Buffer, Buffer*);
+  int32_t (*query_external)(const querier_t*, gas_meter_t*, uint64_t*, Buffer, Buffer*);
 } Querier_vtable;
 
 typedef struct GoQuerier {
+  gas_meter_t *gas_meter;
```

Comment:
Not sure if we need to pass this in (at least for the SDK).
There Querier contains `Context`, which contains `GasMeter`, so the go side already has access to it.

GoQuerier doesn't need any more info. We just need to pass the extra `uint6t_t *used_gas` over the ffi calls

Reply:
done in 5dd0dfd